modules/useQueries.js

```diff
@@ -1,15 +1,22 @@
-import qs from 'qs'
+import warning from 'warning'
+import { parse, stringify } from 'query-string'
 import runTransitionHook from './runTransitionHook'
 import parsePath from './parsePath'
 
 const SEARCH_BASE_KEY = '$searchBase'
 
 function defaultStringifyQuery(query) {
-  return qs.stringify(query, { arrayFormat: 'brackets' }).replace(/%20/g, '+')
+  return stringify(query).replace(/%20/g, '+')
 }
 
-function defaultParseQueryString(queryString) {
-  return qs.parse(queryString.replace(/\+/g, '%20'))
+const defaultParseQueryString = parse
+
+function isNestedObject(object) {
+  for (var p in object)
+    if (typeof object[p] === 'object')
```

Comment:
It looks like it handles those sanely. This is just a developer convenience warning, so they're aware we're not looking to handle nested queries. I wouldn't try to go overboard on supporting a bunch of edge cases here. I think they'll figure it out when they see `%5Bobject+Object%5D` in their URLs :smile: